src/tests/system/tests/test_autofs.py

```diff
@@ -20,9 +20,11 @@
 @pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
 @pytest.mark.require(
     lambda client, sssd_service_user: ((sssd_service_user == "root") or client.features["non-privileged"]),
-    "SSSD was built without support for running under non-root"
+    "SSSD was built without support for running under non-root",
 )
-def test_autofs__cache_first(client: Client, nfs: NFS, provider: GenericProvider, cache_first: bool, sssd_service_user: str):
+def test_autofs__cache_first(
+    client: Client, nfs: NFS, provider: GenericProvider, cache_first: bool, sssd_service_user: str
```

Comment:
Was it also from 'flake8' report?

Reply:
Yes, see error messages pasted bellow. (line too long)